governance-exercises/StableLab-Poll-02/Clive Derecognition as AD - August 9, 2023.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Clive Derecognition as AD - August 9, 2023
+summary: Signal your support or opposition to derecognizing Clive as an AD due to identity compromise based on the materials AD Diane provided.
+discussion_link: https://forum.makerdao.com/
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-08-14 16:00 UTC
```

Comment:
Date doesn't match the title, also the format is wrong, it needs to be: 2023-08-14T16:00:00